src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,22 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```

Comment:
Ensure that Log4j is properly configured in your project to avoid runtime issues related to logging.